packages/@react-spectrum/color/test/ColorArea.test.tsx

```diff
@@ -62,23 +62,25 @@ describe('ColorArea', () => {
     describe('attributes', () => {
       it('sets input props', () => {
         let {getAllByRole} = render(<Component defaultValue={'#ff00ff'} />);
-        let sliders = getAllByRole('slider');
-
-        let [xSlider, ySlider] = sliders;
+        let [xSlider] = getAllByRole('slider');
+        let ySlider = xSlider.nextElementSibling;
```

Comment:
This is fine for now, but a fyi: if you wanna get the aria-hidden slider you can do `getAllByRole('slider', {hidden: true})` as per https://testing-library.com/docs/queries/byrole/#hidden. Would make it less brittle in case the DOM structure changed in the future and the other slider wasn't a sibling of the first slider

Reply:
See 13133361ad96325b1d3b90770e22c2c08e0c8e90.